fieldsolver/gridGlue.cpp

```diff
@@ -388,10 +388,17 @@ void getFieldsFromFsGrid(
          //loop over dccrg cells to which we shall send data for this remoteRank
          auto const &fsgridCells = onFsgridMapCells[dccrgCell];
          for (auto const fsgridCell: fsgridCells){
-         //loop over fsgrid cells for which we compute the average that is sent to dccrgCell on rank remoteRank
-//        if(technicalGrid.get(fsgridCell)->sysBoundaryFlag == sysboundarytype::DO_NOT_COMPUTE) {
-//           continue;
-//        }
+            //loop over fsgrid cells for which we compute the average that is sent to dccrgCell on rank remoteRank
+            if(technicalGrid.get(fsgridCell)->sysBoundaryFlag == sysboundarytype::OUTER_BOUNDARY_PADDING) {
```

Comment:
Oh dear. Another can-of-worms-idea. How will this impact time-varying boundaries? Should we in fact ensure the fields get updated and communicated there as well, so that the volB grows in sync with the actual boundary?

Reply:
They will be as we only skip padding here?

Reply:
Yep, I think so too. Isn't this already handled properly with the introduction of padding?

Reply:
Yeah I guess it'll be fine.